### PR TITLE
Update tutorial for transforms and pretrained weights

### PR DESCRIPTION
**Pull request by hfangcat**
Update tutorial for transforms according to https://github.com/microsoft/torchgeo/issues/2418

### CODE REVIEW

**Comment by hfangcat** [comment]
@microsoft-github-policy-service agree company="KTH"

**Comment by hfangcat** [comment]
Update tutorial for pretrained weights according to https://github.com/microsoft/torchgeo/issues/2418

File: docs/tutorials/transforms.ipynb [hunk 1]
```diff
@@ -429,13 +436,39 @@
...
+    "- **AppendNBR**\n",
+    "- **AppendNDBI**\n",
+    "- **AppendNDRE**\n",
+    "- **AppendNDSI**\n",
+    "- **AppendNDVI**\n",
+    "- **AppendNDWI**\n",
+    "- **AppendNormalizedDifferenceIndex**\n",
+    "- **AppendRBNDVI**\n",
+    "- **AppendSWI**\n",
+    "- **AppendTriBandNormalizedDifferenceIndex**\n",
```
Comment (adamjstewart) [inline docs/tutorials/transforms.ipynb:460]: Not sure if it's worth it to list all of these out. I would instead list a couple indices that TorchGeo knows how to compute and then link to the full API docs.

File: docs/tutorials/transforms.ipynb [hunk 2]
```diff
@@ -500,7 +533,11 @@
     "id": "w4ZbjxPyHoiB"
    },
    "source": [
-    "It's even possible to chain indices along with augmentations from Kornia for a single callable during training."
+    "It's even possible to chain indices along with augmentations from Kornia for a single callable during training.\n",
+    "\n",
+    "Check out the documentation of Kornia's augmentations [here](https://kornia.readthedocs.io/en/latest/augmentation.html).\n",
```
Comment (adamjstewart) [inline docs/tutorials/transforms.ipynb:538]: Let's add an Additional Reading section at the end like I did in my Intro tutorials

File: docs/tutorials/transforms.ipynb (hunk 1 above)
Comment (hfangcat) [reply docs/tutorials/transforms.ipynb]: done!

File: docs/tutorials/pretrained_weights.ipynb [hunk 3]
```diff
@@ -19,6 +19,8 @@
    "source": [
     "# Pretrained Weights\n",
     "\n",
+    "_Written by: Adam J. Stewart and Heng Fang_\n",
```
Comment (adamjstewart) [inline docs/tutorials/pretrained_weights.ipynb:22]: I'm just going to stick with the original author here which is @nilsleh

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**